openerp/openupgrade/doc/source/development.rst

```diff
@@ -1,7 +1,43 @@
+=========================================================
 Migration script development
-++++++++++++++++++++++++++++
+=========================================================
 
 .. toctree::
    :maxdepth: 2
 
    devfaq
+
```

Comment:
This is the index of the document. But it does not refer to the page contents. Instead, it refers to another page and then the document contains other contents not listed in the index. I'm thinking the document layout make more sense if you would your text in a new file called guidelines.rst and add *guidelines* under (or above) *devfaq*. Or you drop the toctree entirely and combine your text with the devfaq.rst file into this file, given that the faq is quite short still.